content/doc/book/security/access-control/permissions.adoc

```diff
@@ -107,6 +105,95 @@ Anything they cannot accomplish through the existing UI they can do through the
 An instance on which an untrusted user gained _Overall/Administer_ permission should be considered fully compromised and should be replaced.
 All secrets (credentials, etc.) stored on that instance should be rotated or revoked and all artifacts created from that point on should be verified.
 
+
+=== Agent permissions ===
+
+==== Agent/Build ====
```

Comment:
I would prefer to not have the trailing `===` characters here so that this remains consistent with the rest of the file.  However, that is just a preference, since it renders correctly in the [preview page](https://deploy-preview-5534--jenkins-io-site-pr.netlify.app/doc/book/security/access-control/permissions/#agent-permissions) already
```suggestion
=== Agent permissions

==== Agent/Build
```